proteus/mprans/BoundaryConditions.py

```diff
@@ -131,8 +131,10 @@ def setNoSlip(self):
         self.w_dirichlet.setConstantBC(0.)
         self.p_advective.setConstantBC(0.)
         self.vof_advective.setConstantBC(0.)
-        self.k_dirichlet.setConstantBC(0.)
-        self.dissipation_diffusive.setConstantBC(0.)
+
```

Comment:
@adimako actually I moved them before because was easier for checking only the turbulent boundaries. Now I re-ordered back to dirichlet conditions first, then advective and diffusive ones.